solutions/test.js

```diff
@@ -0,0 +1,3 @@
+//this is a test file
```

Comment:
remove this empty file from your commit, does this algorithm already have a test written for it?